schema/snapcraft.yaml

```diff
@@ -168,6 +168,15 @@ properties:
             enum:
               - simple
               - forking
+          restart-condition:
+              type: string
+              enum:
+                - never
```

Comment:
Which at least according to [this](https://www.freedesktop.org/software/systemd/man/systemd.service.html) is "no," not "never." Right? Or am I reading something wrong?